apps/music/test/marionette/Player_test.js

```diff
@@ -131,6 +136,36 @@ marionette('Music player tests', function() {
       music.tapHeaderActionButton();
       music.checkPlayerIconShown(true);
     });
+
+    test('Check the play icon is in the status bar', function() {
```

Comment:
This seems different enough from the other tests in this suite that you could consider creating a new suite for this test.  I don't feel strongly about it, though.